pfcpiface/conn.go

```diff
@@ -88,6 +93,45 @@ func (node *PFCPNode) NewPFCPConn(lAddr, rAddr string) *PFCPConn {
 		shutdown:       make(chan struct{}),
 		InstrumentPFCP: node.metrics,
 	}
+
+	p.setLocalNodeID(node.upf.nodeID)
+
+	if buf != nil {
+		// TODO: Check if the first msg is Association Setup Request
+		p.HandlePFCPMsg(buf)
+	}
+
+	// Update map of connections
+	node.pConns[rAddr] = p
+
+	go p.Serve()
+
+	return p
```

Comment:
can remove return val.

Reply:
we need this if we are initiating the connection. So we can call send association on this returned value